api/src/main/java/io/opencensus/trace/export/SpanData.java

```diff
@@ -69,6 +72,7 @@
    *     Span} is still active.
    * @return a new immutable {@code SpanData}.
    */
+  @SuppressWarnings({"deprecation", "unchecked"})
```

Comment:
It would be safer to limit `@SuppressWarnings("unchecked")` to one line by assigning the cast value to a local variable and annotating only the variable.